CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ endif()
 
 # resolving dependencies for the project
 message (STATUS "PROJECT ............................... " ${PROJECT_NAME})
+message (STATUS "CMAKE_VERSION ......................... " ${CMAKE_VERSION})
```

Comment:
Do we really need it? CMake version shouldn't depend on OpenVINO 😄

Reply:
It's just for information about build system